lib/workers/repository/update/pr/changelog/release-notes.ts

```diff
@@ -25,16 +26,14 @@ export async function getReleaseList(
   logger.trace('getReleaseList()');
   const { apiBaseUrl, repository, type } = project;
   try {
-    switch (type) {
-      case 'gitlab':
-        return await gitlab.getReleaseList(apiBaseUrl, repository);
-      case 'github':
-        return await github.getReleaseList(apiBaseUrl, repository);
-
-      default:
-        logger.warn({ apiBaseUrl, repository, type }, 'Invalid project type');
-        return [];
+    if (apiBaseUrl && type === 'gitlab') {
```

Comment:
please leave the switch, i like to add more platforms in near future .

just return null if API url is missing